code/modules/mob/status_procs.dm

```diff
@@ -30,6 +30,8 @@
 			if(stat == CONSCIOUS || stat == SOFT_CRIT)
 				throw_alert("blind", /obj/screen/alert/blind)
 			overlay_fullscreen("blind", /obj/screen/fullscreen/blind)
+			// You are blind why should you be able to make out details like color, only shapes near you
+			src.add_client_colour(/datum/client_colour/monochrome/blind)
```

Comment:
Remove the vestigal src in front of the proc calls and this is good to go.

Reply:
Should be fixed up now